release/schedule/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Release Schedule
+---
+
+WPE WebKit follows a 6-month development cycle, which means that:
+
+- Two major releases are done every year, typically in March and October.
+- Between major releases, any number of minor releases containing bug fixes
+  may be done.
+- Development releases can be produced at any time, of which release
+  candidates are done on the weeks prior to the next major release.
+
+Due to the amount of code shared between [WebKitGTK](https://webkitgtk.org)
+and WPE WebKit, major releases for both products are done simultaneously and
+using same release branches. Note that even when the release teams for both
+projects coordinate to roll out major releases the same dates, keeping minor
+version numbers in sync is done in a best-effort basis.
+
```

Comment:
[WebKitGTK](https://webkitgtk.org) and WPE WebKit share a fair amount of code.
Therefore, both projects release their major versions simultaneously, and share the
same release branches. For minor versions, the release teams of both projects
try to sync their version numbers in a best-effort basis.